test/test_dynamics.jl

```diff
@@ -720,10 +720,13 @@ timesteppers = (:QuasiAdamsBashforth2, :RungeKutta3)
         end
     end
 
+    # This test alone runs for 2 hours on the GPU!!!!! (I Remove it for now...)
```

Comment:
Is this comment still relevant?

Reply:
yes, I removed that test for the moment because it is taking more than 1 hour (if you see it starts at 23.36 and ends at 1.01)

![Screen Shot 2022-07-07 at 3 53 38 PM](https://user-images.githubusercontent.com/33547697/177860556-d5152441-98d1-432e-9735-3f11ce44baf2.png)

Reply:
Can we document this in an issue so we don't forget it?

I was also puzzled many times why this takes so long to finish. Is it compilation time?

Perhaps instead of commenting out the test do `@test_skip`?

Reply:
I guess so, it looks like it is a 0D simulation... 
Anyways, documenting it in an issue is definitely a better course of action

Reply:
how is the syntax of `@test_skip` if I want to skip it only on the GPU?